xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -1084,6 +1084,40 @@ There are some running programs that might use files deleted by recent upgrade.
      The deployer node is now ready to be used to upgrade the remaining
      cloud nodes and running services.
    </para>
+   <warning>
+     <para>
+      If upgrading from Helion OpenStack 8, there is a manual file update that must
+      be applied before continuing the upgrade process. In the file
+      <literal>/usr/share/ardana/ansible/roles/osconfig/tasks/check-product-status.yml</literal>
+      replace `command` with `shell` in the first ansible entry. The correct version
+      of the file appears below.
```

Comment:
We probably should include the blurb about git commit'ing the change, after making it, as well ;-) Though if you try and run ready-deployment.yml as the next step with will complain anyway.

Reply:
thats good feedback, I'll fix it and upload another version, thanks!